enhancements/sig-policy/54-core-policy-module/README.md

```diff
@@ -0,0 +1,227 @@
+# Core Policy Module
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [website](https://github.com/open-cluster-management-io/open-cluster-management-io.github.io/)
+
+## Summary
+
+A common repository / module will contain common API validation, structs, and
+methods that multiple policy controllers can use. This will simplify creating
+new controllers, and assist in maintenance of existing ones. This repository
+will also describe the policy-framework behavior for developers, since the
```

Comment:
Keep this scoped to generic statements about policy controllers. Anything more in depth can be on the website a page focused about the architecture of the Policy Addon.